src/Microsoft.PowerShell.MarkdownRender/VT100EscapeSequences.cs

```diff
@@ -204,6 +205,8 @@ public MarkdownOptionInfo()
         private const string Header5Dark = "[4;96m";
         private const string Header6Dark = "[4;97m";
         private const string CodeDark = "[48;2;155;155;155;38;2;30;30;30m";
+
+        private const string CodeMacOS = "[107;95m";
```

Comment:
Should add a comment why we are special casing macOS

Reply:
Added comment.